may-2018-hardfork.md

```diff
@@ -0,0 +1,33 @@
+# March 2018 Hardfork Specification
```

Comment:
Is this March or May Hardfork Specification ?

Reply:
May.  Thanks